client/GameForm.js

```diff
@@ -9,12 +9,25 @@ import TextField from '@material-ui/core/TextField/TextField';
 import { useInput } from './hooks/InputHook';
 
 const GameForm = ({ history }) => {
-    const {value: listType, bind: bindListType} = useInput('collection');
-    const {value: listOption, bind: bindListOption} = useInput('');
-    const {value: numPlayers, bind: bindNumPlayers} = useInput('');
-    const {value: minDuration, bind: bindMinDuration} = useInput('');
-    const {value: maxDuration, bind: bindMaxDuration} = useInput('');
-   
+    const { value: listType, bind: bindListType } = useInput('collection');
+    const { value: listOption, bind: bindListOption } = useInput('');
+    const { value: numPlayers, bind: bindNumPlayers } = useInput('');
+    const { value: minDuration, bind: bindMinDuration } = useInput('');
+    const { value: maxDuration, bind: bindMaxDuration } = useInput('');
+
+    const calculateMinMax = (defaultMinutes, duration) => {
+        if(duration){
+            return duration;
+        }else{
+            return defaultMinutes;
+        }

```

Comment:
Javascript has a much terser and tidier way of capturing this sort of logic:
```
return duration ? duration : defaultMinutes;
```

Otherwise looks good

Reply:
most languages have ternaries @tjforryan https://en.wikipedia.org/wiki/Ternary_operation

Reply:
I was aware many other languages have ternaries, but I didn't want to make assumptions about familiarity.

I wasn't trying to suggest JS has any special claim on them, but just that it's a useful tool it makes available